### ISSUE TITLE
Trailing '/' on endpoints

### ISSUE THREAD

**Issue by pop** [osuosl/timesync-frontend-flask#92]
There are a few (all?) endpoints which don't work with a trailing slash. Adding an **optional** trailing slash to all endpoints would really make things easier to navigate.

**Comment by subnomo** [comment]
Doing this in Flask is easy, just add the trailing slash in the route decorator, like so:

``` python
@app.route('/login/', methods=['GET', 'POST'])
```

This will also redirect users from `/login` to `/login/`.

@ElijahCaine 
However, I came across a reddit comment I thought was interesting:

>  Let's say you're developing a website and you include an About section.
> 
> I would include a trailing slash if I were to make it possible to view individual bios of members (or would be expected to in the new future).
> 
> Something like:
> 
> ``` python
> @app.route('/about/', defaults={'member':None})
> @app.route('/about/<member>')
> def about(member):
>     pass
> ```
> 
> If I wasn't going to include the option to view individual profiles, I wouldn't include a trailing slash.
> 
> ``` python
> @app.route('/about')
> def about():
>     pass
> ```
> 
> The reasoning I use is the former is not an absolute endpoint. Where as the latter is.

[Source](https://www.reddit.com/r/flask/comments/1rdebk/do_you_include_a_trailing_slash_when_creating/cdm5bx1)

So for example, we would have a trailing slash for `/times/`, but not for `/times/create`.

I'm probably just thinking about this too much, but what do you think?

**Comment by pop** [comment]
@subnomo I see your point (and by extension that reddit post's point). The deepest endpoints don't have a trailing slash but intermediate ones do.

That makes sense, but my one problem with it is the classic 'theory vs practice' remark. In theory that makes sense since you're packing information into your url and that's useful to the user. In practice though I want to go to `whatever.com/ext/` and get to the same page as `whatever.com/ext` would take me. I'm lazy and don't want to think about urls nor trailing slashes.

**tldr** it's a cool idea but I would just add the feature on all endpoints for convenience.
